docs/reference/authentication.md

```diff
@@ -28,13 +28,13 @@ Web UI at Administration / Users to create users.  Users have an access key
 to log into the Web UI, or to authenticate programmatic requests to the API
 Server or the S3 Gateway.
 
-#### lakeFS Invite
+#### LakeFS Invite
```

Comment:
Please revert this change, per @adipolak 's already-mentioned [https://docs.google.com/document/d/1vif05nDiKZYPwYdINLoq4BJ6j7bBIQ6ZA2dt7McuzyY/](guide).  The spelling "lakeFS" is explicitly mentioned there:

> Make sure to spell the brand name correctly: **lakeFS**. Not LakeFS or lakefs.

The first word on the website lakefs.io is "lakeFS".  If you go to @peacing 's [latest blog](https://lakefs.io/towards-effective-dataops/), you can find "lakeFS" at the start of a sentence, _in a heading_.

The inconsistent capitalization in the heading is on the _second_ word.  This document is consistent in using a single capital at the start of every heading at level 4.  And AFAICS this is a consistent practice on our documentation.  The correct heading is therefore

```suggestion
#### lakeFS invite
```

**BUT:** Please just leave it alone and pull this PR.  We can fix it later.

Thanks!